packages/component/src/chip/Chip.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+	ChipProps,
+	ColorTokens,
+	CornerRadiusTokens,
+	FontFamilyTokens,
+	FontSizeTokens,
+	IconSizeTokens,
+	IconTokens,
+	SpaceTypeTokens,
+	StackAlignType,
+	StackType,
+} from '@blue-learn/schema';
+import React, { memo } from 'react';
+import { View, Text } from 'react-native';
+import ThemeProvider from '@blue-learn/theme';
+import Typography from '../typography/Typography';
+import Icon from '../icon/Icon';
+import Space from '../space/Space';
+import Stack from '../stack/Stack';
+
+/**
+ * @description
+ * Stack supports children as React Component and widgetItems as WidgetItem Type[order by children -> widgetItems]
+ * Note: renderItem should be passed to render WidgetItem
+ **/
+const Chip: React.FC<ChipProps> = ({
+	label = 'Chip',
+	bgColor,
+	labelColor = ColorTokens.Grey_100,
+	fontSize = FontSizeTokens['2xs'],
+	borderRadius = CornerRadiusTokens.BR4,
+	iconAlignment = 'left',
+	icon,
+	padding,
+	paddingHorizontal = SpaceTypeTokens.MD,
+	paddingVertical = SpaceTypeTokens.SM,
+}) => {
+	const theme = ThemeProvider.getTheme();
+
+	const borderRadiusValue =
+		theme.borderRadius[borderRadius];
+
+	const backgroundColor = theme.colors[bgColor];
+
+	const paddingTop =
+		theme.space[paddingVertical || padding?.top];
+
+	const paddingBottom =
+		theme.space[paddingVertical || padding?.bottom];
+
+	const paddingLeft =
+		theme.space[paddingHorizontal || padding?.left];
+
+	const paddingRight =
+		theme.space[
+			paddingHorizontal || padding?.right
+		];
+
+	return (
+		<Stack>
```

Comment:
what is the use of stack here?
i think view can be replaced by stack

Reply:
okay i will change that Stack to View